src/lib/select/OVERVIEW.md

```diff
@@ -1,8 +1,41 @@
 `<md-select>` is a form control for selecting a value from a set of options, similar to the native
-`<select>` element.
+`<select>` element. You can read more about selects in the 
+[Material Design spec](https://material.google.com/components/menus.html).
 
-<!-- example(select-overview) -->
```

Comment:
This comment needs to stay. This is what embeds the live example on the docs site.